src/hotspot/cpu/x86/templateTable_x86.cpp

```diff
@@ -3987,9 +4015,9 @@ void TemplateTable::invokedynamic(int byte_no) {
 
   load_invokedynamic_entry(rbx_method);
   // rax: CallSite object (from cpool->resolved_references[f1])
-  // rbx: MH.linkToCallSite method (from f2)
+  // rbx: MH.linkToCallSite method
```

Comment:
I'm not sure why you edited these comments in the x86 code but not in the aarch64 code. To avoid issues for maintainers it is better to make the changes in both rather than just one.